src/git-istage/Shortcuts.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+
+namespace GitIStage
+{
+    internal class Shortcuts
```

Comment:
I don't see this class being used anywhere, is this a remnant of a time before you added descriptions to console commands?